app/scripts/models/account.js

```diff
@@ -16,31 +16,41 @@ define([
   'lib/constants',
   'models/profile-image'
 ], function (Backbone, _, p, AuthErrors, ProfileClient, Constants, ProfileImage) {
-
   // Account attributes that can be persisted
   var PERSISTENT = {
-    uid: undefined,
+    accessToken: undefined,
     email: undefined,
+    grantedPermissions: undefined,
+    lastLogin: undefined,
+    needsOptedInToMarketingEmail: undefined,
+    profileImageId: undefined,
+    profileImageUrl: undefined,
     sessionToken: undefined,
     sessionTokenContext: undefined,
-    accessToken: undefined,
-    verified: undefined,
-    profileImageUrl: undefined,
-    profileImageId: undefined,
-    lastLogin: undefined,
-    grantedPermissions: undefined
+    uid: undefined,
+    verified: undefined
   };
 
   var DEFAULTS = _.extend({
-    password: undefined,
-    unwrapBKey: undefined,
+    customizeSync: undefined,
     keyFetchToken: undefined,
-    customizeSync: undefined
+    password: undefined,
+    unwrapBKey: undefined
   }, PERSISTENT);
 
   var ALLOWED_KEYS = Object.keys(DEFAULTS);
   var ALLOWED_PERSISTENT_KEYS = Object.keys(PERSISTENT);
 
+  function withMarketingEmailClient(method) {
+    return function () {
+      var self = this;
+      return self.getOAuthTokenFromSessionToken(self.get('sessionToken'), 'basket')
```

Comment:
> (It's not just about cognitive load either - one concrete downside of making these all methods on the Account, is that we must load the code for all of them even if the user never sees anything to do with their profile or email marketing prefs. Something to think about in the future if we ever want to chunk up our downloads a little more).

I had a similar concern when putting together this patch. app-start.js creates several clients that are passed down through user.js to account.js, or passed into views. Except for fxa-js, none of these clients keep any state AFAICT, though they rely on information stored in the URLs returned from a call to the `/config` endpoint. It seems to me a more natural approach would be to pass around a "ServiceURLs" or "Configuration" object into User/Account, which could then create the clients as needed, reducing the number of objects instantiated on starutp.